whylog/tests/tests_teacher/test_rule_validation.py

```diff
@@ -1,24 +1,37 @@
 from whylog.config.filename_matchers import WildCardFilenameMatcher
 from whylog.config.log_type import LogType
-from whylog.tests.tests_teacher import TestBase
+from whylog.constraints import TimeConstraint
+from whylog.tests.tests_teacher import TestBase, TestEffectBase, TestRuleBase
 
 from whylog.teacher.rule_validation_problems import (  # isort:skip
     NoEffectParserProblem, NotSetLogTypeProblem, NotUniqueParserNameProblem, ParserCountProblem
 )  # yapf: disable
+from whylog.constraints.validation_problems import (  # isort:skip
+    MinGreaterThatMaxProblem, ParamConversionProblem
+)
 
 
 class TestValidationBase(TestBase):
+    def _in_parser_problems(self, parser_id, problem):
```

Comment:
Rename it to `_check_if_parser_has_problems`. The name of the function should be verb. Function do something and that verb tells what does the function do. Function without verbs in the name are unclear.